x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/expanded_row_field_header/expanded_row_field_header.tsx

```diff
@@ -9,7 +9,12 @@ import { EuiText } from '@elastic/eui';
 import React from 'react';
 
 export const ExpandedRowFieldHeader = ({ children }: { children: React.ReactNode }) => (
-  <EuiText size="xs" color={'subdued'} className={'fieldDataCard__valuesTitle'}>
+  <EuiText
+    size="xs"
+    color={'subdued'}
+    className={'fieldDataCard__valuesTitle'}
```

Comment:
Nit - I think this class name must have been created when this was part of a `FieldDataCard` component, but might be better to rename it now to correspond to it being in the `expanded_row_field_header` of the stats table.

Reply:
Updated these in the latest changes